content-services/7.2/develop/repo-ext-points/content-transformers-renditions.md

```diff
@@ -218,8 +218,7 @@ changed by resetting the following Alfresco global property.
 local.transform.pipeline.config.dir=shared/classes/alfresco/extension/transform/pipelines
 ```
 
-Local transform-core-aio docker image might need a [newer version of image and approach](#adding-pipelines-and-failover-transforms-to-a-t-engine)
-with providing config directly to the T-Engine.
+Direct override is only possible in 3.0.0 version of alfresco-transform-core and higher. To override local transform-core-aio T-Engine configuration, the docker image needs a newer version of the image and follow this approach providing configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).
```

Comment:
The meaning is severely changed. The problem with transform-core-aio is that it loads the configuration of other t-engines in a static manner, while you can override the configuration of other specific t-engines (like tika, libreoffice), the transform-core-aio (all-in-one) in particular has issues with overrides due to how it loads configs of all t-engines grouped beneath it.

"To override local transform-core-aio T-Engine configuration, the docker image needs to be version 3.0.0 or higher and following approach of providing configuration directly to the T-Engine should be used:  [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine)."

That could be used, but I wouldn't mention alfresco-transform-core here, it's a base project for all t-engines, shared file store, t-router, where there is no issue (albeit it's way harder than the new approach).